4.0.1/docker-compose.yml

```diff
@@ -4,11 +4,73 @@ volumes:
   geonetwork:
   esdata:
   pgdata:
+  pglog:
+  nginxlog:
+  apachelog:
+
+
+networks:
+  gn-network:
+    driver: bridge
+
 
 services:
+  www:
+    image: nginx
+    ports:
+      - 80:80
+    environment:
+      # Only used for / redirect to default webapp
+      - NGINX_PUBLIC_HOST=localhost
+      #- NGINX_PUBLIC_HOST=apps.titellus.net
+      - NGINX_HOST=0.0.0.0
+      - NGINX_PORT=80
+    volumes:
+      - ./nginx:/etc/nginx/templates
+      - nginxlog:/var/log/nginx
+    depends_on:
+      - geonetwork
+    networks:
+      - gn-network
+
+#  www2:
```

Comment:
Ok it's unused, following previous comment, is it for documentation purposes ?

Reply:
Yep, some projects want to use apache.